sections/video.liquid

```diff
@@ -0,0 +1,130 @@
+{{ 'video-section.css' | asset_url | stylesheet_tag }}
+{{ 'component-deferred-media.css' | asset_url | stylesheet_tag }}
+
+<div class="video-section{% unless section.settings.full_width %} page-width{% endunless %}">
+  <h2 class="title{% if section.settings.full_width %} page-width{% endif %}">{{ section.settings.heading }}</h2>
+  <div{% if section.settings.image_padding %} class="video-section-padding"{% endif %}>
+    <noscript>
+      <div class="video-section__media"
+        {% if section.settings.cover_image != blank and section.settings.image_padding %} style="padding-bottom: {{ 1 | divided_by: section.settings.cover_image.aspect_ratio | times: 100 }}%;"{% endif %}
+      >
+        <a href="{{ section.settings.video_url }}" class="video-section__poster media media--transparent{% unless section.settings.image_padding %} media--landscape{% endunless %}{% if section.settings.cover_image == blank %} video-section__placeholder{% endif %}">
+          {%- if section.settings.cover_image != blank -%}
+            <img
+              srcset="{%- if section.settings.cover_image.width >= 500 -%}{{ section.settings.cover_image | img_url: '500x' }} 500w,{%- endif -%}
+              {%- if section.settings.cover_image.width >= 730 -%}{{ section.settings.cover_image | img_url: '730x' }} 730w,{%- endif -%}
+              {%- if section.settings.cover_image.width >= 1440 -%}{{ section.settings.cover_image | img_url: '1440x' }} 1440w,{%- endif -%}
+              {%- if section.settings.cover_image.width >= 990 -%}{{ section.settings.cover_image | img_url: '990x' }} 990w,{%- endif -%}
+              {%- if section.settings.cover_image.width >= 2080 -%}{{ section.settings.cover_image | img_url: '2080x' }} 2080w{%- endif -%}"
+              src="{{ section.settings.cover_image | img_url: '533x' }}"
+              sizes="(min-width: 990px)990px, (min-width: 750px) calc(100vw - 100px), calc(100vw - 30px)"
+              alt="{{ section.settings.description | escape }}"
+              loading="lazy"
+              width="{{ section.settings.cover_image.width }}"
+              height="{{ section.settings.cover_image.height }}"
+              class="video-section__image"
```

Comment:
I think we can remove all instances of `.video-section__image` because I don't see it being used. What are your thoughts?